drivers/CMakeLists.txt

```diff
@@ -26,6 +25,22 @@ if(CMAKE_COMPILER_IS_MSVC)
     endif()
 endif()
 
+if(TF_PSA_CRYPTO_TEST_DRIVER)
+    # Use `libtestdriver1` as the driver name to align with the name used in
+    # Mbed TLS. This allows to reuse most of the test driver code as it is.
```

Comment:
I think that's in reference to Mbed TLS 3.6, right? In context I would understand it as referring to the version of Mbed TLS that matches the version of TF-PSA-Crypto that I'm reading, but I don't see why the content of that version of Mbed TLS would matter here.

Reply:
I've tried and improved the comment.